### ISSUE TITLE
Teensy 3.5 3.6 Digital Write

### ISSUE THREAD

**Issue by lacan** [bonnom/Arduino32BitBoards#5]
Hello

I am trying to wrap my head around the code for the Teensy 3.5/3.6 Board
https://github.com/bonnom/Arduino32BitBoards/blob/master/Firmwares/Teensy35_36_Arduino32bitBoard_Firmware/Teensy35_36_Arduino32bitBoard_Firmware.ino

When in MM I would like to send a TTL signal, which pins should I use? It seems all pins are set to PWM and I should set them first to 100% power and then send the sequence, is that correct?

**Comment by bonnom** [comment]
That is right, If you set power to 100% it should be always high. You can use any of the PWM for that signal.
You might need to check with an oscilloscope with if it truly is always and not unexpectedly sometimes be low while it is at 100%
